util/reggen/field.py

```diff
@@ -328,3 +377,30 @@ def sw_readable(self) -> bool:
 
     def sw_writable(self) -> bool:
         return self.swaccess.key != 'ro'
+
+    def apply_alias(self, alias_field: object, where: str) -> None:
+        '''
+        Compare all attributes and replace overridable values.
```

Comment:
Nit: I think the usual format is for the 1st line to go on the same line as the `'''`

Reply:
Will fix.